pkg/morph/client/neofs/wrapper/cheque.go

```diff
@@ -0,0 +1,16 @@
+package neofscontract
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+// Cheque invokes `cheque` method of NeoFS contract.
+func (x *ClientWrapper) Cheque(id []byte, user util.Uint160, amount int64, lock util.Uint160) error {
+	return x.Cheque(id, user, amount, lock)
```

Comment:
Linter says it should be 
```suggestion
	return (*neofscontract.Client)(x).Cheque(id, user, amount, lock)
```